content/community/courses.md

```diff
@@ -28,7 +28,7 @@ permalink: community/courses.html
 
 - [Egghead.io](https://egghead.io/browse/frameworks/react) - Short instructional videos on React and many other topics.
 
-- [Frontend Masters](https://frontendmasters.com/courses/) - Video courses on React and other frontend frameworks.
+- [Frontend Masters](https://frontendmasters.com/learn/react/) - Video courses on React and other frontend frameworks.
```

Comment:
Considering the text after the link, it looks like the original link was intended. Maybe check when this was added to see if there was discussion about whether it should link to anything frontend related or just React.

Reply:
Ok noted...I was trying to replicate the egghead.io link considering that maybe the exact path to react was not yet added to the Frontend Masters platform by then

Reply:
Maybe back then they didn't have a react specific section so the description was adjusted accordingly. We should now update the link description to e.g 
```suggestion
- [Frontend Masters](https://frontendmasters.com/learn/react/) - Video courses on React.
```

Reply:
Looks good...I've committed the changes as suggested